src/frontend/utils/amplitude.ts

```diff
@@ -22,76 +22,49 @@ export function logEvent(eventName: string, eventProperties: any) {
     amplitudeInstance.logEvent(eventName, eventProperties);
 }
 
-export const logSidevisningBarnetrygd = (side: string) => {
+export const logSidevisningBarnetrygd = (side: string, søknadstype: ESøknadstype) => {
     logEvent('sidevisning', {
         side,
         team_id: 'familie',
-        skjemanavn: søknadstyper[hentSøknadstype()].navn,
-        skjemaId: søknadstyper[hentSøknadstype()].id,
+        skjemanavn: søknadstyper[søknadstype].navn,
+        skjemaId: søknadstyper[søknadstype].id,
     });
 };
 
-export const logSkjemaStartet = () => {
+export const logSkjemaStartet = (søknadstype: ESøknadstype) => {
     logEvent('skjema startet', {
-        skjemanavn: søknadstyper[hentSøknadstype()].navn,
-        skjemaId: søknadstyper[hentSøknadstype()].id,
+        skjemanavn: søknadstyper[søknadstype].navn,
+        skjemaId: søknadstyper[søknadstype].id,
         team_id: 'familie',
     });
 };
 
-export const logForsettPåSøknad = () => {
+export const logFortsettPåSøknad = (søknadstype: ESøknadstype) => {
     logEvent('fortsett på søknad', {
-        skjemanavn: søknadstyper[hentSøknadstype()].navn,
-        skjemaId: søknadstyper[hentSøknadstype()].id,
+        skjemanavn: søknadstyper[søknadstype].navn,
+        skjemaId: søknadstyper[søknadstype].id,
         team_id: 'familie',
     });
 };
 
-export const logSkjemaStegFullført = (steg: number) => {
+export const logSkjemaStegFullført = (steg: number, søknadstype: ESøknadstype) => {
     logEvent('skjemasteg fullført', {
-        skjemanavn: søknadstyper[hentSøknadstype()].navn,
-        skjemaId: søknadstyper[hentSøknadstype()].id,
+        skjemanavn: søknadstyper[søknadstype].navn,
+        skjemaId: søknadstyper[søknadstype].id,
         team_id: 'familie',
         steg,
     });
 };
 
-export const logKlikkGåVidere = (steg: number) => {
+export const logKlikkGåVidere = (steg: number, søknadstype: ESøknadstype) => {
     logEvent('klikk gå videre', {
-        skjemanavn: søknadstyper[hentSøknadstype()].navn,
-        skjemaId: søknadstyper[hentSøknadstype()].id,
+        skjemanavn: søknadstyper[søknadstype].navn,
+        skjemaId: søknadstyper[søknadstype].id,
         team_id: 'familie',
         steg,
     });
 };
 
-export const logSpørsmålBesvart = (spørsmålSpråktekstId: string, svar: string) => {
```

Comment:
Tidligere har vi fulgt med på hvor mange som faller av på spørsmålet om folkeregistrert adresse. Vi bruker ikke dette lenger og fjerner kallet heller enn å gjøre den omskrivingen som kreves for å fortsette å støtte dette